runtime/parachains/src/inclusion/mod.rs

```diff
@@ -791,6 +950,92 @@ impl<T: Config> Pallet<T> {
 			)
 	}
 
+	/// Check that all the upward messages sent by a candidate pass the acceptance criteria. Returns
+	/// false, if any of the messages doesn't pass.
+	// FAIL-CI: TODO update docs
```

Comment:
Is this still relevant?

Reply:
Yes, still have to update it :laughing:   
Wanted to wait for the code review first, but will do soon.